core/config/project_settings.cpp

```diff
@@ -788,7 +788,7 @@ Error ProjectSettings::_save_settings_text(const String &p_file, const Map<Strin
 	}
 	file->store_string("\n");
 
-	for (Map<String, List<String>>::Element *E = props.front(); E; E = E->next()) {
+	for (const Map<String, List<String>>::Element *E = props.front(); E; E = E->next()) {
```

Comment:
Looks like this one was missed?

Reply:
No, it was intentional, because of `E != props.front()`